src/rpc/common/impl/HandlerProvider.hpp

```diff
@@ -41,6 +42,9 @@ class Counters;
 namespace feed {
 class SubscriptionManager;
 }  // namespace feed
+namespace data {
+class AmendmentCenter;
```

Comment:
Do you need forwarding declaration here? It seems there is only `AmendmentCenterInterface` used in this file.